README.md

```diff
@@ -2,4 +2,196 @@
 
 [![Build Status](https://travis-ci.com/sevenwestmedia-labs/typescript-object-validator.svg?branch=master)](https://travis-ci.com/sevenwestmedia-labs/typescript-object-validator)
 
-A simple library for typescript projects for validating object shapes.
+A simple library for typescript projects to validating object shapes. You can use this package to declare the shape you'd like an object to align to. This is handy when you have a complex object (for example, an API response) and want to validate the object, AND have typescript recognize the shape.
```

Comment:
The first sentence here has a typo/odd word usage.
```diff
- A simple library for typescript projects to validating object shapes.
+ A simple library for typescript projects for validating object shapes. 
```
(the double for in the diff suggestion reads a bit weird, so you might want to do something different 😉 )